python/src/DistributionImplementation_doc.i.in

```diff
@@ -821,7 +821,9 @@ Only valid for univariate and bivariate distributions.
 
 See Also
 --------
-computeCDF, viewer.View, ResourceMap
+openturns.Distribution.computeCDF
+openturns.viewer.View
+openturns.ResourceMap
```

Comment:
I would completely remove the "See also" section here.

Reply:
lets do that in another PR